modules/apps/dispatch/dispatch-web/src/main/resources/META-INF/resources/view_dispatch_trigger.jsp

```diff
@@ -75,20 +75,10 @@ PortletURL portletURL = PortletURLBuilder.create(
 						property="createDate"
 					/>
 
-					<%
-					DispatchTriggerMetadata dispatchTriggerMetadata = dispatchTriggerDisplayContext.getDispatchTriggerMetadata(dispatchTrigger.getDispatchTriggerId());
-
-					String nextFireDateString = LanguageUtil.get(request, "not-scheduled");
-
-					if (dispatchTriggerMetadata.isDispatchTaskExecutorReady() && (dispatchTrigger.getNextFireDate() != null)) {
-						nextFireDateString = fastDateTimeFormat.format(dispatchTrigger.getNextFireDate());
-					}
-					%>
-
 					<liferay-ui:search-container-column-text
 						cssClass="table-cell-ws-nowrap"
 						name="next-fire-date"
-						value="<%= nextFireDateString %>"
+						value="<%= dispatchTriggerDisplayContext.getNextFireDateString(dispatchTrigger) %>"
```

Comment:
@alessiorendina can you review my change here? The Next Run Date was always display in UTC. I changed it to display in the dispatch trigger's time zone, so it's easier to verify the scheduled time set in the cron expression. 

<img width="1254" alt="Screen Shot 2024-02-29 at 6 08 37 PM" src="https://github.com/liferay-publications/liferay-portal/assets/5168351/75f8b03e-d83c-4118-aef7-92407418fb69">
<img width="1204" alt="Screen Shot 2024-02-29 at 6 09 08 PM" src="https://github.com/liferay-publications/liferay-portal/assets/5168351/5a749d48-993a-491d-ae38-5e04262fe8dc">